igvc/launch/camera.launch

```diff
@@ -1,26 +1,42 @@
 <!-- camera.launch -->
```

Comment:
This file should not exist in this pull request

Reply:
It's in the title of the PR, but if you don't want it I can throw it out.

Reply:
the camera.launch file was already condensed.

Reply:
The file is still here